app/services/sso/jwt.rb

```diff
@@ -35,5 +35,9 @@ def jwt_token_from_request
     def bearer_token_set?
       request.authorization.present? && request.authorization.start_with?('Bearer')
     end
+
+    def no_issuer?
```

Comment:
I know @tbrisker suggested this, but the `no_` prefix brings us into multiple negatives. How about
```suggestion
    def missing_issuer?
```

Reply:
I feel that `no_issuer?` is a indicative enough name - the issuer is not present.

`missing_issuer` indicates that we were expecting an issuer, but the issuer is missing, which is not the case here. We expect that there is no issuer in the token.

 @tbrisker @ekohl  Let me know your thoughts!

Reply:
I agree, we are expecting there to be no issuer on the token to differentiate oidc from general jwt. @ekohl unless you feel very strongly about this I'm fine with it as is.

Reply:
No, I don't feel strongly about this and I think I agree with the argument. It's also a private method so it's not that important. If does become an issue, refactoring should easy anyway.